Sources/ForageSDK/ForageSDK.swift

```diff
@@ -28,10 +28,6 @@ public class ForageSDK {
             assertionFailure("ForageSDK is not initialized - call ForageSDK.setup() before accessing ForageSDK.shared")
             return
         }
-        environment = Environment(sessionToken: config.sessionToken)
```

Comment:
`updateSessionToken` inside of `setup` updates the environment. The new unit tests will help catch any regressions/issues around environment-setting!